tests/errors/MultiInstMeasures.hs

```diff
@@ -1,4 +1,4 @@
-{-@ LIQUID "--expect-error-containing=Multiple instance measures `sizeOf` for type `GHC.Internal.Ptr.Ptr`" @-}
+{-@ LIQUID "--expect-error-containing=Multiple instance measures `MultiInstMeasures.sizeOf` for type `GHC.Internal.Ptr.Ptr`" @-}
```

Comment:
I would not expect a qualified name for an identifier from the current module.

Reply:
It is an artifact of renaming. Names are qualified now, and I'm guessing the tidy passes for errors would need to be updated to strip the qualifiers or replace them with import aliases.

I'm leaving that for another time, but feel free to object.